Packs/GitHub/Integrations/GitHub/GitHub_test.py

```diff
@@ -74,6 +74,33 @@ def mock_http_request(method, url_suffix, params=None, data=None, headers=None,
 ]
 
 
+RETURN_ERROR_TARGET = 'GitHub.return_error'
+
+
+@pytest.mark.parametrize('params, expected_result', [
+    ({'credentials': {'password': '1234'}}, "Insert api token or private key"),
+    ({'sshkey': 'sshkey'}, "Insert api token or private key")
+])
+def test_params(mocker, params, expected_result):
+    """
+    Given:
+      - Configuration parameters
+    When:
+      - One of the required parameters are missed.
+    Then:
+      - Ensure the exception message as expected.
+    """
+
+    mocker.patch.object(demisto, 'params', return_value=params)
+    return_error_mock = mocker.patch(RETURN_ERROR_TARGET)
```

Comment:
Why did you set RETURN_ERROR_TARGET as constant?

Reply:
I saw that on other integration's tests and thought it's a better practice, for future uses.